### DISCUSSION TITLE
Audio files wont drag and drop

### DISCUSSION THREAD

**Discussion by gatd** [apohl79/audiogridder#1281] [Help]
Audio files wont drag and drop to plugin that allows it so cant load a new sample

**Comment by apohl79** [comment]
This is not supported

**Comment by gatd** [comment]
Hi
I can not get multi outputs to show in audio bidder plugin in logic pro mojave
just mono stereo and 5.1 are my choices

my friends show the multi ’s out as an option
any help please 
thanks



Simon Slater
***@***.***
